testing/apt-dater-host/APKBUILD

```diff
@@ -0,0 +1,26 @@
+# Contributor: Henrik Riomar <henrik.riomar@gmail.com>
+# Maintainer:
```

Comment:
Yes, I am the upstream author of the Alpine implementation of apt-dater-host, so that is no problem, I can maintain the package i Alpine.